templates/homepage.html.twig

```diff
@@ -34,15 +35,43 @@
                         </v-list-item>
                     </v-list>
                 </v-navigation-drawer>
+
                 <v-app-bar app>
                     <v-app-bar-nav-icon @click.stop="drawer = !drawer" v-if="hasDrawer"></v-app-bar-nav-icon>
                     <v-toolbar-title><v-icon>bear-it</v-icon> BearIT</v-toolbar-title>
                     <div class="flex-grow-1"></div>
-                    <v-avatar v-if="loggedIn">
-                        <img :src="user.avatar" :alt="user.login" :title="user.name" />
-                    </v-avatar>
+                    <v-menu bottom :offset-y="true" v-if="hasDrawer">
```

Comment:
If the props is boolean type you can just add `offset-y` : `<v-menu bottom offset-y v-if="hasDrawer">`. If the argument is not present it will be false.

Reply:
good to know, thanks for feedback.